lua/user/plugins.lua

```diff
@@ -381,51 +379,49 @@ M.config = function()
         }
       end,
       event = "BufRead",
-      disable = lvim.builtin.smooth_scroll ~= "cinnamon",
+      enable = lvim.builtin.smooth_scroll == "cinnamon",
```

Comment:
shouldn't this be `enabled` instead of `enable`?

Reply:
Fixed in f090f8a